packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphComponent/Tooltip.tsx

```diff
@@ -269,13 +329,19 @@ function formatDouble(
   const leftPercent = ratioToPercent(leftRatio);
   const rightPercent = ratioToPercent(rightRatio);
 
-  const left = `Baseline: ${numberWithCommas(
-    totalLeft
-  )} samples, ${formatter.format(totalLeft, sampleRate)} (${leftPercent}%)`;
+  const newLeft = {
+    percent: leftPercent,
```

Comment:
I think this is a bug that was already happening but I got a `NaN` here 

https://user-images.githubusercontent.com/6951209/178542964-38e01418-5890-4b29-b5f0-0a6505e38240.mp4

Reply:
To reproduce use something like this 
![image](https://user-images.githubusercontent.com/6951209/178544411-176905aa-5431-4bce-80d4-6837faaa343e.png)